src/portable/renesas/usba/dcd_usba.c

```diff
@@ -485,13 +518,22 @@ static void process_set_address(uint8_t rhport)
 {
   const uint32_t addr = USB0.USBADDR.BIT.USBADDR;
   if (!addr) return;
+#if defined(__CCRX__)
+  tusb_control_request_t setup_packet;
```

Comment:
oh, ccrx does not support this style of initialization ? Hmm, there is a lots more of this within codebase.

Reply:
This is the style of the C99 standard. CCRX supports the C99 standard by specifying `-lang=c99` as a command line argument. Since TinyUSB implicitly requires c99, I think CCRX should also have this option.

Reply:
yeah c99 is really old now, it should be enabled to build with.

Reply:
Don't worry. I found the solution that also works with GCC, so the check which toolchain is used is no longer required. The problem with this struct definition and CCRX is, that one of the struct member is defined as an union which includes some bit fields (that could be an error of the CCRX toolchain ? ). 

BTW: I already use C99 language option.